stix_shifter/stix_transmission/src/modules/aws_cloud_watch_logs/aws_cloud_watch_logs_results_connector.py

```diff
@@ -0,0 +1,157 @@
+from ..base.base_results_connector import BaseResultsConnector
+import json
+from flatten_json import flatten
+import copy
+from .....utils.error_response import ErrorResponder
+from os import path
+
+MAX_LIMIT = 1000
+
+
+class InvalidParameterException(Exception):
+    pass
+
+
+class AWSCloudWatchLogsResultsConnector(BaseResultsConnector):
+    def __init__(self, client):
+        self.client = client
+
+    def create_results_connection(self, search_id, offset, length):
+        """
+        Fetching the results using search id, offset and length
+        :param search_id: str, search id generated in transmit query
+        :param offset: str, offset value
+        :param length: str, length value
+        :return: dict
+        """
+        return_obj = dict()
+        response_dict = dict()
+        try:
+            query = dict()
+            offset = int(offset)
+            length = int(length)
+            total_records = offset+length
+            if total_records <= MAX_LIMIT:
+                query['queryId'] = search_id
+                response_dict = self.client.get_query_results(**query)
+                return_obj['success'] = True
+                results = response_dict['results'][offset:total_records]
+                result_list = []
+                guardduty_common_attr = self.get_guardduty_common_attr()
```

Comment:
Is it necessary to pass the contents of `guardduty_common_attr` from function to function? i see its only used in `process_flatten_guardduty_results`. you can just directly call `self.get_guardduty_common_attr()` inside `process_flatten_guardduty_results` if you dont have any other checks before that.

Reply:
We will look into it

Reply:
Done